test/unit/memoCtrl.test.js

```diff
@@ -128,7 +102,7 @@ describe('Contol functions for combining data', () => {
 
   test('Merge kopps data and api data, memo api has empty values', async () => {
     const newKoppsData = await memoCtrl.mergeKoppsCourseAndMemoData(
-      mockedKoppsTemplates,
+      JSON.parse(JSON.stringify(mockedKoppsTemplates)),
```

Comment:
If the above suggestions are applied, this parsing wont be needed.

```suggestion
mockedKoppsTemplates,
```